pages.zh/linux/ul.md

```diff
@@ -0,0 +1,13 @@
+# ul
+
+> 执行文本的下划线。
+> 给定字符串中的每个字符必须单独加下划线。
+> 更多说明请访问： <https://manned.org/ul>。
```

Comment:
```suggestion
> 更多说明请访问： <https://manned.org/ul>.
```
We use half-width punctuation afer Latin characters

Reply:
had fixed